README.md

```diff
@@ -1,2 +1,62 @@
-# gtm-consent-mode-examples
-Google Tag Manager Consent Mode Examples
+# About This Template
```

Comment:
Since this is the home page for the project, I would supply a descriptive title like:
```
# Google Tag Manager Consent Mode Examples
```